.changeset/clever-cats-jump.md

```diff
@@ -0,0 +1,10 @@
+---
+"fluid-framework": minor
+"@fluidframework/map": minor
```

Comment:
There was a discussion about this on another PR, I'm on the side that we should try to be deliberate about this bit in a changeset. What we're doing here qualifies as a major change.

Reply:
oops it is automatically merged in, I will create a small PR to update it.